google-cloud-storage/src/test/java/com/google/cloud/storage/testing/ApiPolicyComparatorTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.storage.testing;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import com.google.api.services.storage.model.Policy;
+import com.google.api.services.storage.model.Policy.Bindings;
+import com.google.common.collect.ImmutableList;
+
+public class ApiPolicyComparatorTest {
```

Comment:
If we are going to use a comparator, we should test the not-equals case, and also test that if `compare(p1,p2)==-1` then `compare(p2,p1)==1`